core/src/main/java/org/jboss/pnc/build/finder/core/Checksum.java

```diff
@@ -258,6 +277,14 @@ public void setFilename(String filename) {
         this.filename = filename;
     }
 
+    public long getFileSize() {
+        return fileSize;
+    }
+
+    public void setFileSize(long fileSize) {
+        this.fileSize = fileSize;
+    }
+
     @Override
     public String toString() {
```

Comment:
Regenerate `toString()` to include `fileSize`.

Reply:
Sure!